en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-260-to-400.md

```diff
@@ -2224,6 +2224,11 @@ Follow the instructions below to migrate existing API Manager resources from the
         ALTER TABLE AM_API_RATINGS
             ADD RATING_ID VARCHAR(255) DEFAULT (SYS_GUID()) NOT NULL
         /
+        
+        ALTER TABLE AM_API_RATINGS
+            ADD CONSTRAINT add_pk PRIMARY KEY (RATING_ID)
```

Comment:
Let's add a different name to this constraint. This should be unique. (add_pk)

Reply:
Changed this to add_pk_ratings.